File: Java/GreatestOfThree.java
import java.util.Scanner;
public class GreatestOfThree {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.println("Enter three numbers");
            int a=sc.nextInt();
            int b=sc.nextInt();
            int c=sc.nextInt();
            if (a > b && a > c) {
                System.out.println(a + " is the greatest number");
            } else if (b > a && b > c) {
                System.out.println(b + " is the greatest number");
            }else if (c > a && c > b) {
                System.out.println(c + " is the greatest number");
            } else {
                System.out.println("all numbers are equal");
            }
        }
    }
}

File: c/GreatestOfThree.c
#include<stdio.h>
void  main(){
    int a,b,c;
    printf("Enter three numbers\n");
    scanf("%d%d%d",&a,&b,&c);
    if(a>b && a>c){
        printf("%d is greatest",a);
    }
    else if(b>a && b>c){
        printf("%d is greatest",b);
    }
    else if(c>b &&c>b){
        printf("%d is greatest",c);
    }
    else{
        printf("all numbers are equAL");
    }
}